Aflores plymouth (#234)
osisoft/Edge-Data-Store-Docs: content/modbus/shared-content/content/configuration/security.md

---
uid: PIAdapterForModbusSecurity
---

# Security

> When determining Modbus TCP security practices with regard to REST APIs, you should consider the following practice. To keep the adapter secure, only administrators should have access to machines where the adapter is installed. REST APIs are bound to localhost, meaning that only requests coming from within the machine are accepted.

Reviewer: ```suggestion
When determining Modbus TCP security practices with regard to REST APIs, only administrators should have access to machines where the adapter is installed to keep the adapter secure. REST APIs are bound to localhost, which means that only requests originating from within the machine are accepted.
```
